common/src/Renderer/BrushRendererArrays.h

```diff
@@ -69,26 +70,30 @@ namespace TrenchBroom {
         template<typename T>
         class VboBlockHolder {
```

Comment:
Should this not be renamed to VboHolder now?

Reply:
Yes will fix